Tests/AfluentTests/WorkerTests/ShareTests.swift

```diff
@@ -12,8 +12,8 @@ import Testing
 
 struct ShareTests {
     @Test func unsharedTaskExecutesRepeatedly() async throws {
-        try await withMainSerialExecutor {
-            try await confirmation(expectedCount: 4) { exp in
+        try await confirmation(expectedCount: 4) { exp in
```

Comment:
https://github.com/pointfreeco/swift-concurrency-extras/issues/27

Reply:
[It looks like they merged the fix for this](https://github.com/pointfreeco/swift-concurrency-extras/pull/28)! Not sure if you want to pull in a SHA to get it, but just wanted to update the thread.